_posts/2015-09-03-flink-forward.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title:  "Announcing Flink Forward 2015"
+date:   2015-09-03 08:00:00
+categories: news
+---
+
+[Flink Forward 2015](http://flink-forward.org/) is the first
+conference with Flink at its center that aims to bring together the
+Apache Flink community in a single place. The organizers are starting
+this conference in October 12 and 13 from Berlin, the place where
+Apache Flink started.
+
+<center>
+<img src="{{ site.baseurl }}/img/blog/flink-forward-banner.png" style="width:80%;margin:15px">
+</center>
+
+The [conference program](http://flink-forward.org/?post_type=day) has
+been announced by the organizers and a program committee consisting of
+Flink PMC members. The agenda contains talks from industry and
+academia as well as a dedicated session on hands-on Flink training.
+
+Some highlights of the talks include
+
+- [William
+  Vambenepe](http://flink-forward.org/?speaker=william-vambenepe),
+  lead of the product management team responsible for Big Data
+  services on Google Cloud Platform (BigQuery, Dataflow, etc...) on
+  data streaming, Google Cloud Dataflow, and Apache Flink.
+
+- Talks by several practitioners on how they are putting
+  Flink to work in their projects.
+
+- Talks on how open source projects, including Apache Mahout, Apache
+  SAMOA (incubating), Apache Zeppelin (incubating), Apache BigTop, and
+  Apache Storm integrate with Apache Flink.
+
+- Talks by Flink committers on several aspects of the system, such as
+  fault tolerance, the internal runtime architecture, and others.
+
+Check out the schedule [here](http://flink-forward.org/?post_type=day)
```

Comment:
Might be personal taste but I'm opposed to `here` links. There are also SEO reasons to avoid them.
> Check out the [schedule](http://flink-forward.org/?post_type=day).